nixos/modules/programs/wayland/niri.nix

```diff
@@ -53,6 +53,33 @@ in
           # https://github.com/YaLTeR/niri/wiki/Important-Software#portals
           extraPortals = [ pkgs.xdg-desktop-portal-gnome ];
         };
+
+        # Recommended for Nvidia GPU's
+        # https://github.com/YaLTeR/niri/wiki/Important-
```

Comment:
Broken link? Should probably be <https://github.com/YaLTeR/niri/wiki/Nvidia#high-vram-usage-fix> or <https://yalter.github.io/niri/Nvidia.html#high-vram-usage-fix>